subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -62,6 +62,19 @@ config BT_EATT_AUTO_CONNECT
 
 endif # BT_EATT
 
+config BT_GATT_AUTO_RESUBSCRIBE
+	bool "Automatic re-subscription to characteristics."
```

Comment:
minor issue: remove the `.` at the end. No other Kconfig option seem to use formal sentence termination for the description this way